tools/alerting.go

```diff
@@ -179,7 +181,81 @@ var GetAlertRuleByUID = mcpgrafana.MustTool(
 	getAlertRuleByUID,
 )
 
+type ListContactPointsParams struct {
+	Limit int     `json:"limit,omitempty" jsonschema:"description=The maximum number of results to return. Default is 100."`
+	Name  *string `json:"name,omitempty" jsonschema:"description=Filter contact points by name"`
+}
+
+func (p ListContactPointsParams) validate() error {
+	if p.Limit < 0 {
+		return fmt.Errorf("invalid limit: %d, must be greater than 0", p.Limit)
+	}
+	return nil
+}
+
+type contactPointSummary struct {
+	UID  string  `json:"uid"`
+	Name string  `json:"name"`
+	Type *string `json:"type,omitempty"`
+}
+
+func listContactPoints(ctx context.Context, args ListContactPointsParams) ([]contactPointSummary, error) {
+	if err := args.validate(); err != nil {
+		return nil, fmt.Errorf("list contact points: %w", err)
+	}
+
+	c := mcpgrafana.GrafanaClientFromContext(ctx)
+
+	params := provisioning.NewGetContactpointsParams().WithContext(ctx)
+	if args.Name != nil {
+		params.Name = args.Name
+	}
+
+	response, err := c.Provisioning.GetContactpoints(params)
+	if err != nil {
+		return nil, fmt.Errorf("list contact points: %w", err)
+	}
+
+	pagedContactPoints, err := applyLimitToContactPoints(response.Payload, args.Limit)
```

Comment:
```suggestion
	filteredContactPoints, err := applyLimitToContactPoints(response.Payload, args.Limit)
```
? or something similar?